client/a8c-for-agencies/sections/sites/sites-dashboard/index.tsx

```diff
@@ -266,6 +264,12 @@ export function SitesDashboard() {
 					) }
 				</LayoutTop>
 
+				<div className="sites-dashboard__notices">
```

Comment:
Since we are not using the LayoutBody here, we are displaying the notices manually.